src/Umbraco.Web/PublishedCache/NuCache/ContentStore.cs

```diff
@@ -352,7 +359,7 @@ public void UpdateContentTypes(IEnumerable<int> removedIds, IEnumerable<IPublish
                     var node = link.Value;
                     if (node == null) continue;
                     var contentTypeId = node.ContentType.Id;
-                    if (removedIdsA.Contains(contentTypeId)) removedContentTypeNodes.Add(node.Id);
+                    if (removedIds.Contains(contentTypeId)) removedContentTypeNodes.Add(node.Id);
```

Comment:
Possible null reference exception
```suggestion
                    if (removedIdsA.Contains(contentTypeId)) removedContentTypeNodes.Add(node.Id);
```